FR/2DCode/raw/test_pcr_ok.json

```diff
@@ -25,16 +25,16 @@
     }
```

Comment:
Value of ci should be dgci:V1:FR:QSMFF6WU45QA0F0XTD2HS8WPO:10

Reply:
fix ok

Reply:
1. Value of ci in JSON does not match with CBOR / COSE payload.
2. Value of 'dr' and 'sc' under 't' are still int. They should be date value in ISO format.

![image](https://user-images.githubusercontent.com/84303320/118824916-d3321700-b8ec-11eb-9a2a-640a0451b47f.png)